COMMUNITY.md

```diff
@@ -106,8 +106,8 @@ For people listed below we made exceptions:
 | Name | Slack handle | Github handle | Affiliation |
 | -- | -- | -- | -- |
 | Stacey Potter   | `@Stacey Potter` | `@staceypotter` | Dynatrace  |
-| Tamao Nakahara  | `@tamao`         | `@mewzherder`   | Weaveworks |
-| Kingdon Barrett | `@KingdonB`      | `@kingdonb`     | Weaveworks |
+| Tamao Nakahara  | `@tamao`         | `@mewzherder`   | Independent |
+| Kingdon Barrett | `@KingdonB`      | `@kingdonb`     | Independent |
```

Comment:
@kingdonb this should go into a diffrent PR. Also the commits to this PR should be made Tamao and signed by her.

Reply:
I think we tried using the GitHub UI and it went horribly wrong. I'll confer with Tamao and put the community affiliation changes in a separate PR 👍

Reply:
I put the affiliation changes for COMMUNITY.md into #345 - they are separate now

We will still need to add @mewzherder sign-off and signature, (or is only sign-off enough?)

You should be able to add your sign-off with `git commit --amend --sign-off` so you get authorship on the commit, or you could close this PR and try a new one. I don't know if you want to try `git pull --rebase` -- it might just work?

Reply:
I removed my sign-off so the DCO bot shows the check missing properly, so it's clearer what we are talking about.

I think it will be easier for Tamao to delete the fork and start over like you say 👍